artifacts/src/main/java/org/openmbee/sdvc/artifacts/json/ArtifactJson.java

```diff
@@ -0,0 +1,95 @@
+package org.openmbee.sdvc.artifacts.json;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+import org.openmbee.sdvc.json.ElementJson;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Schema(name = "Artifact")
+@JsonIgnoreProperties({"empty"})
+public class ArtifactJson extends HashMap<String, Object> {
+
+    public static final String ARTIFACTS = "_artifacts";
+    public static final String MIMETYPE = "mimetype";
+    public static final String EXTENSION = "extension";
+    public static final String LOCATION = "location";
+    public static final String LOCATIONTYPE = "locationType";
+
+    public ArtifactJson() {
+        super();
+    }
+
+    public ArtifactJson(Map<String, Object> map) {
+        super(map);
+    }
+
+    @JsonProperty(MIMETYPE)
+    @Schema(accessMode = Schema.AccessMode.READ_ONLY)
+    public String getMimeType() {
+        return (String) this.get(MIMETYPE);
+    }
+
+    @JsonProperty(MIMETYPE)
+    public ArtifactJson setMimeType(String mimeType) {
+        this.put(MIMETYPE, mimeType);
+        return  this;
+    }
+
+    @JsonProperty(EXTENSION)
+    @Schema(accessMode = Schema.AccessMode.READ_ONLY)
+    public String getExtension() {
+        return (String) this.get(EXTENSION);
+    }
+
+    @JsonProperty(EXTENSION)
+    public ArtifactJson setExtension(String extension) {
+        this.put(EXTENSION, extension);
+        return  this;
+    }
+
+    @JsonProperty(LOCATION)
+    @Schema(accessMode = Schema.AccessMode.READ_ONLY)
+    public String getLocation() {
+        return (String) this.get(LOCATION);
+    }
+
+    @JsonProperty(LOCATION)
+    public ArtifactJson setLocation(String location) {
+        this.put(LOCATION, location);
+        return  this;
+    }
+
+    @JsonProperty(LOCATIONTYPE)
+    @Schema(accessMode = Schema.AccessMode.READ_ONLY)
+    public String getLocationType() {
+        return (String) this.get(LOCATIONTYPE);
+    }
+
+    @JsonProperty(LOCATIONTYPE)
+    public ArtifactJson setLocationType(String locationType) {
+        this.put(LOCATIONTYPE, locationType);
+        return  this;
+    }
+
+    public static List<ArtifactJson> getArtifacts(ElementJson elementJson){
+
+        List<Object> rawArtifacts = (List)elementJson.get(ARTIFACTS);
+        if(rawArtifacts == null || rawArtifacts.isEmpty()) {
+            return null;
```

Comment:
prefer return empty list instead of null, so caller doesn't have to check for nulls